docs/source/platform/operation-registry.md

```diff
@@ -1,55 +1,58 @@
+- Provide demand control for their production GraphQL APIs.
+- Permit the exact operations necessary for their client applications.
+- Eliminate the risk of unexpected, and possibly costly, operations being executed against their graph.
 
-## Getting started
+Operations defined within client applications can be extracted and uploaded to Apollo Engine using the Apollo CLI. Apollo Server then fetches a manifest of these operations from Apollo Engine and forbids the execution of any operations that were not in that manifest.
 
-### Prerequisites
+## Prerequisites
 
-* Apollo Server 2.2.x (or newer).
-  * Subscriptions should be disabled when using the operation registry.  For more information, see the instructions below.  Please contact the Apollo sales team if this support is necessary.
-  * To get started with Apollo Server, visit [its documentation](/docs/apollo-server/).
-* A client application which utilizes `gql` tagged template literals for its operations or, alternatively, stores operations in `.graphql` files.
-* An Apollo Engine API key.
-  * To obtain an API key, visit [Apollo Engine](https://engine.apollographql.com) and create a service.
+Setting up operation registration and safelisting is a full-stack process, so you will need to have access to the client code containing the operations you want to register, and to the server within which you want to enforce the safelist.
 
-### Installation steps
+1. **Apollo Server `2.2.x`** (or newer). Subscriptions should be disabled when using the operation registry. For more information, see the instructions below. Please [contact the Apollo sales team](https://www.apollographql.com/contact-sales/) if this support is necessary. To get started with Apollo Server, visit [its documentation](/docs/apollo-server/).
+1. A **client application** which utilizes `gql` tagged template literals for its operations or, alternatively, stores operations in `.graphql` files.
+1. An **Apollo Engine API key**. To obtain an API key, visit [Apollo Engine](https://engine.apollographql.com) and create a service. You will need and Engine account with a Team or Enterprise subscription to be able to upload to the operation registry.
 
-> Make sure you've met the requirements for _Prerequisites_ above.
+## Installation steps
 
 These installation steps require access to both the client and server codebases to perform the following tasks:
 
-* The `apollo` CLI is used to search the client codebase for GraphQL operations and upload them to Apollo Engine.
-* Apollo Server is then configured with a plugin which fetches the manifest from Apollo Server and enforces safe-listing using that manifest.
+- The **Apollo CLI** is used to search the client codebase for GraphQL operations and upload them to Apollo Engine.
+- **Apollo Server** is then configured with a plugin which fetches the manifest from Apollo Engine and enforces safelisting using that manifest.
```

Comment:
This is a fantastic point. I think we should keep it "safelisting" here because that's how we've been spelling it internally, so it will remain consistent, even if it's incorrect.

It's a larger task to move every instance of the word "safelisting" in the Apollo domains to "safe-listing". This spans to other articles and our marketing pages, at least. I'm in favor of that and think it should be done, but we should do it all at once and separately. I don't think we should have inconsistencies where they can be avoided, even though I'm in favor of moving towards "safe-listing" everywhere.

Reply:
Hmm, it's only on four locations of our entire site including the blog.  I could make all the changes in about 3 minutes if you agree! 😄